client/libra-dev/src/event.rs

```diff
@@ -117,7 +117,7 @@ pub unsafe extern "C" fn libra_LibraEvent_from(
                 receiver_address: addr,
                 amount: res.amount(),
                 metadata: metadata_ptr,
-                metadata_len,
+                metadata_len: metadata_len.try_into().unwrap(),
```

Comment:
Not sure if LibraPaymentEvent is an internal only structure, and is safe to change to use a usize, rather than a u64.  Probably wont mater for a decade, but seems dirty.

Reply:
Please use https://github.com/rust-lang/rust-bindgen/pull/1720/files#diff-d0250c128ebebfe222162f7ea1d5b9e0R1 the new usize = size_t commandline option instead of doing a conversion.

Reply:
Handled.   Took me a minute to realize I needed the quote 1.0.2 -> quote 1.0.3 as well.   Plus some reading up in size_t.